ModuleAnimatedAnchoredDecoupler.cs

```diff
@@ -211,11 +211,19 @@ public EventData<float> OnStop
 				return OnStoppedEvent;
 			}
 		}
-		
-		//
-		// Methods
-		//
-		public bool IsMoving ()
+
+        public string ScalarModuleID
+        {
+            get
+            {
+                throw new NotImplementedException();

```

Comment:
Thanks for the code contribution but I have grave misgivings about the throwing of exceptions in this manner. Especially in a function where a string ID is being requested. This isn't a good practice in a multimod sandbox that we all have to play in and if this function throws at the wrong time it could halt execution of other people's code.

I can't accept this pull request with this change present.